static/app/utils/performance/quickTrace/utils.tsx

```diff
@@ -208,6 +213,10 @@ export function parseQuickTrace(
     }
   });
 
+  if (isFullTrace && projects.size > 1) {
+    handleProjectMeta(organization, projects.size);
```

Comment:
Are we only interested in the number of unique organizations with connected services? Because capturing the analytics in this method may result in over-counting since this is called every time the quick trace component is rendered, which can happen more than once per page load.

Reply:
So amplitude will let us group by organization_id so duplicate events shouldn't be an issue here